src/discover.ts

```diff
@@ -112,6 +112,7 @@ export class TestFileDiscoverer extends vscode.Disposable {
             workspaceFolder.uri,
             include,
           )
+          const workspacePath = workspaceFolder.uri.fsPath
```

Comment:
This is a drive-by fix as this was showing an error due to the missing variable when the refresh button was clicked.